packages/wonder-blocks-form/src/util/types.ts

```diff
@@ -9,7 +9,7 @@ import Choice from "../components/choice";
 // Shared props for radio-core and checkbox-core
 export type ChoiceCoreProps = AriaProps & {
     /** Whether this component is checked */
-    checked: boolean;
+    checked: boolean | null | undefined;
```

Comment:
**thought:** Giving a second thought about from where we should export `Checked`... maybe this is the best place from where we should define the `Checked` type as it is the shared/central place for both checkboxes and radiobuttons. wdyt?

Reply:
I agree! That makes more sense to me as well. Especially since we're using the same type for radio